nixos/modules/services/desktops/gnome3/gnome-settings-daemon.nix

```diff
@@ -40,6 +40,8 @@ in
 
     services.udev.packages = [ cfg.package ];
 
+    security.wrappers.gsd-backlight-helper.source = "${pkgs.gnome3.gnome-settings-daemon}/libexec/gsd-backlight-helper";
```

Comment:
I'm getting this when running the wrapper:

>❯ gsd-backlight-helper 
This program can only be used by the root user

So we're hitting this for some reason: https://gitlab.gnome.org/GNOME/gnome-settings-daemon/blob/master/plugins/power/gsd-backlight-helper.c#L65

Reply:
Is this resolved with the latest version of this PR?

Reply:
I've left this alone for a bit because we just need to get 3.32 on master first.
Then debugging would be quick.

Though I'm not sure why we're getting https://gitlab.gnome.org/GNOME/gnome-settings-daemon/blob/master/plugins/power/gsd-backlight-helper.c#L65